### PR TITLE
added ioBanker to list of exchanges

### PR DESCRIPTION
**Pull request by Agorise**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,7 +186,8 @@ do your own research!
 
 * [Citadel](https://citadel.li/)
 * [GDEX](https://www.gdex.io/) 
-* [Rudex](https://rudex.org/)
+* [ioBanker](https://iobanker.com/) - Stablecoins too! (HONEST.BTC, .CNY, .ETH, .USD, .XAG, .XAU, .XRP) 
+* [RuDex](https://rudex.org/)
```
Comment (abitmore) [inline README.md:190]: `RuDEX`.

File: README.md [hunk 2]
```diff
@@ -186,7 +186,8 @@ do your own research!
 
 * [Citadel](https://citadel.li/)
 * [GDEX](https://www.gdex.io/) 
-* [Rudex](https://rudex.org/)
+* [ioBanker](https://iobanker.com/) - Stablecoins too! (HONEST.BTC, .CNY, .ETH, .USD, .XAG, .XAU, .XRP)
```
Comment (abitmore) [inline README.md:189]: Why specially mention stablecoins for ioBanker? Other exchanges don't support stablecoins?

Comment (froooze) [reply README.md:189]: Because ioBanker is now the only UI which features HONEST.Assets out of the box.

Comment (abitmore) [reply README.md:189]: Featured tokens of other UIs aren't listed in the page, IMO we either list for all or for none.

File: README.md [hunk 3]
```diff
@@ -184,11 +184,12 @@ do your own research!
...
-* [GDEX](https://www.gdex.io/) 
-* [Rudex](https://rudex.org/)
-* [WALLDEX](https://w.walldex.pro/) 
-* [XBTS](https://xbts.io/)
+* [Citadel](https://citadel.li/) - Stablecoins too! (bitCNY, bitEUR, bitGold, bitSilver, bitUSD) 
+* [GDEX](https://www.gdex.io/) - Stablecoins too! (bitCNY, bitEUR, bitGold, bitSilver, bitUSD) 
+* [ioBanker](https://iobanker.com/) - Stablecoins too! (HONEST.BTC, .CNY, .ETH, .USD, .XAG, .XAU, .XRP, etc) 
+* [RuDex](https://rudex.org/) - Stablecoins too! (bitCNY, bitEUR, bitGold, bitSilver, bitUSD) 
+* [WALLDEX](https://w.walldex.pro/) - Stablecoins too! (bitCNY, bitEUR, bitGold, bitSilver, bitUSD) 
+* [XBTS](https://xbts.io/) - Stablecoins too! (bitCNY, bitEUR, bitGold, bitSilver, bitUSD)
```
Comment (abitmore) [inline README.md:193]: Looks hilarious.

Comment (ioBanker) [reply README.md:193]: > Looks hilarious.

I agree :p~

**Pull request merged**